controller.go

```diff
@@ -270,6 +285,21 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 
 	network.processOptions(options...)
 
+	if !c.started {
+		// Global stores are watched only when the controller enters the 'started' state.
```

Comment:
@mrjana valid points. but this was added with some inbuilt assumptions and existing behavior.

1. we dont create default-network (other than docker0) during system init. Default-networks are created lazily during container activation.
2. endpoints are not created during the init phase.
3. Global networks will not be created in this phase (watch isnt enabled yet).